apps/mobile/src/screens/ResultsScreen.js

```diff
@@ -11,18 +12,31 @@ type Props = {|
         +travelTo: string,
         +dateFrom: string,
         +dateTo: string,
+        +returnDateFrom: string,
+        +returnDateTo: string,
       |},
     |},
   |},
 |};
 
 export default class ResultsScreen extends React.Component<Props> {
+  static navigationOptions = ({ navigation }: any) => ({
+    headerTintColor: defaultTokens.paletteProductNormal,
+    headerTitleStyle: { color: defaultTokens.colorTextPrimary },
+    title: `${navigation.getParam('travelFrom')} ${
+      navigation.getParam('dateTo') ? '\u21C4' : '\u2192'
```

Comment:
We could use one of the components we created. I can't remember how it's called, but that should be possible.